src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -339,18 +336,15 @@ def __init__(self, method_name):
         super().__init__(method_name)
 
     # Note for this test to run you subscrition needs to give access to batch https://learn.microsoft.com/azure/batch/batch-account-create-portal#allow-batch-to-access-the-subscription
-    @live_only()
-    @ResourceGroupPreparer(location='eastus')
+    @ResourceGroupPreparer(location='eastus2')
     def test_batch_byos_account_cmd(self, resource_group):
```

Comment:
This test keeps failing in PR's CI and based on my investigation, you may need to update https://github.com/Azure/azure-cli/blob/ef6af5770f1be28f76979414408669b5d454a317/src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py#L336 to
```
class BatchMgmtByosScenarioTests(BatchScenarioMixin,ScenarioTest):
```
Because in this scenario test, data plane commands (`batch pool create`, `batch pool show`) are used, `BatchMgmtScenarioMixin` is not enough, `BatchScenarioMixin` should be used